app/presenters/photo_presenter.rb

```diff
@@ -12,7 +12,8 @@ def base_hash
       sizes:      {
         small:  url(:thumb_small),
         medium: url(:thumb_medium),
-        large:  url(:scaled_full)
+        large:  url(:scaled_full),
+        raw:    url,
```

Comment:
Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.

Reply:
That's what I get for fixing the stylebot, eh?